Tasks/SonarQubePostTest/CodeAnalysisFilePathComputation.ps1

```diff
@@ -189,12 +183,11 @@ function ProcessSonarCodeAnalysisReport
     $sonarReportProcessedRootObj | ConvertTo-Json | Set-Content -Path $sonarReportProcessedFilePath
 }
 
-#creates a mapping of msbuild project guid and the path of .xxproj file on disk using ProjectInfo.xml file
+# Creates a mapping of msbuild project guid and the path of .xxproj file on disk using ProjectInfo.xml file
 function CreateProjectGuidAndPathMap
-{
-    param([string][ValidateNotNullOrEmpty()]$agentBuildDirectory)
-
-    $parentFolder = [System.IO.Path]::Combine($agentBuildDirectory, ".sonarqube", "out")
+{    
+    $sqFolder = GetSonarQubeBuildDirectory
+    $parentFolder = [System.IO.Path]::Combine($sqFolder, "out")
```

Comment:
For consistency and maintainability it would be better to create a function to calculate the location of the "out" directory rather than calculating it inline.